lib/galaxy/model/__init__.py

```diff
@@ -8239,9 +8239,9 @@ def copy_to(self, copied_step, step_mapping, user=None):
                 copied_subworkflow = subworkflow
             else:
                 # Can this even happen, building a workflow with a subworkflow you don't own ?
```

Comment:
I guess this comment could be removed / updated?

Reply:
Indeed, thank you!